### DISCUSSION TITLE
how to create a model

### DISCUSSION THREAD

**Discussion by sazearte** [nomic-ai/pygpt4all#79] [Q&A]
Hello, I can't find any info on the net, I wonder if it is possible to train your own model with your data set?

I'm wondering if it's possible to train your own model with your own data set? Does anyone have a basic example in a jupyter notebook to create a model from a text and export it to a model.bin file?

thanks in advance.

**Comment by absadiki** [comment]
Hello @sazearte,

The dataset and the training process are available on the [gpt4all](https://github.com/nomic-ai/gpt4all) repo. 
This repo is just for CPU inference of the pretrained models.
